components/bl-navigation-dots/README.md

```diff
@@ -0,0 +1,125 @@
+# Navigation Dots
+
+Navigation Dots is the component that can be used in Backendless [UI-Builder](https://backendless.com/developers/#ui-builder). This component provides a visually appealing way to navigate through different sections on a webpage.
+
+<p align="center">
+  <img src="./thumbnail.png" alt="main thumbnail" width="780"/>
+</p>
+
+## Properties
+
+| Property                                    | Type                                           | Default value     | Logic                    | Data Binding | UI Setting | Description                                                                                                               |
+|---------------------------------------------|------------------------------------------------|-------------------|--------------------------|--------------|------------|---------------------------------------------------------------------------------------------------------------------------|
+| Sections <br> `sections`                    | *JSON*                                         | `[]`              | Sections Logic           | YES          | NO         | This logic allows you to specify the array of sections.  Signature of sections: list of DOM elements `[Element, Element]` |
```

Comment:
I'm not clearly understand how we could create dynamic count of sections using data binding?